models/models.js

```diff
@@ -12,3 +12,24 @@ exports.fetchCategories = () => {
     });
   }
 };
+
+exports.fetchReviews = (review_id) => {
+  console.log("in the model");
+  if (isNaN(review_id) === true) {
```

Comment:
This isn't necessary! If the invalid id makes it down to the db query, this will cause an error from PSQL automatically (which is where that '22P02') will come from up in your app.